examples/dashboard.ipynb

```diff
@@ -0,0 +1,139 @@
+   "source": [
+    "The Intake `source` object lets us treat data in many different formats the same in the rest of the code here. We can now build a class that captures some parameters that the user can vary along with how those parameters relate to the code needed to update the displayed plot of that data source:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "gopts = hv.opts.WMTS(width=800, height=500, xaxis=None, yaxis=None, bgcolor='black', show_grid=False)\n",
+    "cmaps = {n: colorcet.palette[n] for n in ['fire', 'bgy', 'bgyw', 'bmy', 'gray', 'kbc']}\n",
+    "norms = {'Histogram_Equalization': 'eq_hist', 'Linear': 'linear', 'Log': 'log', 'Cube root': 'cbrt'}\n",
+    "maps  = ['CartoMidnight', 'StamenWatercolor', 'StamenTonerBackground', 'EsriImagery', 'EsriUSATopo', 'EsriTerrain']\n",
+    "bases = {name: ts.relabel(name) for name, ts in gts.tile_sources.items() if name in maps}\n",
+    "plots = {source.metadata['plots'][p].get('label', p):p for p in source.plots}\n",
+    "logo  = \"https://raw.githubusercontent.com/pyviz/datashader/master/doc/_static/logo_horizontal_s.png\"\n",
+    "\n",
+    "class Explorer(pm.Parameterized):\n",
+    "    plot          = pm.ObjectSelector( precedence=0.10, default=source.plots[0], objects=plots)\n",
+    "    normalization = pm.ObjectSelector( precedence=0.12, default='eq_hist',       objects=norms)\n",
+    "    cmap          = pm.ObjectSelector( precedence=0.14, default=cmaps['fire'],   objects=cmaps)\n",
+    "    spreading     = pm.Integer(        precedence=0.16, default=0, bounds=(0, 5))\n",
+    "    \n",
+    "    basemap       = pm.ObjectSelector( precedence=0.18, default=bases['EsriImagery'], objects=bases)\n",
+    "    data_opacity  = pm.Magnitude(      precedence=0.20, default=1.00, doc=\"Alpha value for the data\")\n",
+    "    map_opacity   = pm.Magnitude(      precedence=0.22, default=0.75, doc=\"Alpha value for the map\")\n",
+    "    show_labels   = pm.Boolean(        precedence=0.24, default=True)\n",
+    "    \n",
+    "    @pm.depends('plot')\n",
+    "    def points(self):\n",
+    "        return getattr(source.plot, self.plot)() # could add extra hvplot kwargs here\n",
+    "\n",
+    "    @pm.depends('map_opacity','basemap')\n",
+    "    def tiles(self):\n",
+    "        return self.basemap.opts(gopts).opts(alpha=self.map_opacity)\n",
+    "\n",
+    "    @pm.depends('show_labels')\n",
+    "    def labels(self):\n",
+    "        return gts.StamenLabels.options(level='annotation', alpha=1 if self.show_labels else 0)\n",
+    "    \n",
+    "    @pm.depends('data_opacity')\n",
+    "    def apply_opacity(self, shaded):\n",
+    "        return shaded.opts(alpha=self.data_opacity, show_legend=False)\n",
+    "\n",
+    "    def view(self,**kwargs):\n",
+    "        points   = hv.DynamicMap(self.points)\n",
+    "        agg      = rasterize(points, width=600, height=400)\n",
```

Comment:
To aggregate on another field, you need to add an aggregator to the rasterize call. The field can be another param stream. 

```suggestion
    "        agg      = rasterize(points, width=600, height=400, aggregator=rd.mean('passenger_count'))\n",
```